content/vault/v1.20.x/content/api-docs/system/storage/raftautosnapshots.mdx

```diff
@@ -138,7 +153,11 @@ parameters in the context of AWS EKS & S3 configuration.
 - `azure_container_name` `(string: <required>)` - Azure container name to write
   snapshots to.
 
-- `azure_account_name` `(string)` - Azure account name.
+- `azure_account_name` `(string)` - Azure account name. Make sure that this account has 
+  the following permissions in Azure Blob Storage, so that Vault can store and manage the snapshots:
```

Comment:
```suggestion
- `azure_account_name` `(string)` - Azure account name. The associated Azure
  account must have the following permissions in Azure Blob Storage so that
  Vault can store and manage the snapshots:
```